packages/astro/src/cli/add/index.ts

```diff
@@ -160,6 +160,7 @@ export async function add(names: string[], { flags }: AddOptions) {
 					possibleConfigFiles: [
 						'./tailwind.config.cjs',
 						'./tailwind.config.mjs',
+						'./tailwind.config.ts',
```

Comment:
Should we also add .mts and .cts here?

Reply:
I think so since we already support cjs and mjs

Reply:
Added!